### ISSUE TITLE
Lack of solution or hints to exercises

### ISSUE THREAD

**Issue by senpl** [cmstead/jsLearnerForms#10]
There should be some way that when user could get solution or at least partial hint to provided forms. Without it when someone stucks it is clueless what to do.

**Comment by cmstead** [comment]
Agreed, I'm trying to figure out the best way to do that. I have been working on videos, but that will take a long time to get through.

I used to have walk throughs for some, but they hard to follow.
